aspnetcore/fundamentals/logging/index.md

```diff
@@ -503,35 +670,33 @@ Each provider defines an *alias* that can be used in configuration in place of t
 
 There's a minimum level setting that takes effect only if no rules from configuration or code apply for a given provider and category. The following example shows how to set the minimum level:
 
-[!code-csharp[](index/samples/2.x/TodoApiSample/Program.cs?name=snippet_MinLevel&highlight=3)]
+::: moniker range=">= aspnetcore-3.0"
 
-If you don't explicitly set the minimum level, the default value is `Information`, which means that `Trace` and `Debug` logs are ignored.
+[!code-csharp[](index/samples/3.x/TodoApiSample/Program.cs?name=snippet_MinLevel&highlight=3)]
 
-### Filter functions
+::: moniker-end
 
-A filter function is invoked for all providers and categories that don't have rules assigned to them by configuration or code. Code in the function has access to the provider type, category, and log level. For example:
+::: moniker range="< aspnetcore-3.0"
 
-[!code-csharp[](index/samples/2.x/TodoApiSample/Program.cs?name=snippet_FilterFunction&highlight=5-13)]
+[!code-csharp[](index/samples/2.x/TodoApiSample/Program.cs?name=snippet_MinLevel&highlight=3)]
 
 ::: moniker-end
 
-::: moniker range="< aspnetcore-2.0"
```

Comment:
Are we supposed to be removing 2.0 (and lower) docs?

Reply:
Yes, actually, on most doc's we're only keeping 2.2 and 3.0 version.

Reply:
Removing 2.1 would be concerning since that's the latest LTS

Reply:
> Removing 2.1 would be concerning since that's the latest LTS

 We've dropped 2.1 and lower on virtually all the docs and have had no customer complaints. We've run this by @danroth27   We just don't have the resources to maintain all the previous versions.
If you feel this document justifies the extra work, we can make an exception.

Reply:
You guys are talking past each other a little bit.

We aren't removing 2.1 ... at least I'm not.

We're removing <=2.0. We're keeping >=2.1.

Reply:
We'll keep the 2.1 version.